src/mainboard/clevo/adl-p/ramstage.c

```diff
@@ -239,7 +283,7 @@ static void mainboard_smbios_strings(struct device *dev, struct smbios_type11 *t
 	if (result == 0) {
 		printk(BIOS_DEBUG, "EC firmware version: %s\n", ec_version);
 		t->count = smbios_add_string(t->eos, strconcat("EC firmware version: ",
-					     ec_version));
+							ec_version));
```

Comment:
Still wrong. Please revert it to how it was before the PR.

Reply:
fixed by ed5634a7b9e8fedd83bc84d662c4baa599bd4029